homeassistant/components/syncthru/sensor.py

```diff
@@ -140,6 +124,11 @@ def device_state_attributes(self):
         """Return the state attributes of the device."""
         return self._attributes
 
+    @property
+    def device_info(self):
+        """Return device information."""
+        return {"connections": device_connections(self.syncthru)}
```

Comment:
Since `device_connections` can return an empty set, it would be safer to set the identifier to link them up.

Reply:
Ah, TIL! Until now I for some reason thought the connections were the only thing available for this. Identifiers seem much better, thanks.